### DISCUSSION TITLE
Remap suspend (ctrl-z) in shells that don't handle it

### DISCUSSION THREAD

**Discussion by Zoybean** [helix-editor/helix#10545] [General]
My daily driver shell (`nushell`) doesn't handle suspended processes, so if I accidentally hit ctrl-z in helix, I'm left unable to use the shell or helix. Well, turns out that it can be remapped like anything else:

```
[keys.normal]
C-z = 'no_op'

[keys.select]
C-z = 'no_op'
```

Figured I'd leave this discussion here for anyone else who is searching for how to disable it.

**Comment by jesusmgg** [comment]
Thank you! C-z will no longer be a 50-50 coin toss between "have I saved in the last 15 minutes?" answers now.
